integration/test_resources/help/cloud-account_iac-generate_azure

```diff
@@ -0,0 +1,53 @@
+Use this command to generate Terraform code for deploying Lacework into new Azure environment.
+
+By default, this command will function interactively, prompting for the required information to setup the new cloud account. In interactive mode, this command will:
+		
+* Prompt for the required information to setup the integration
+* Generate new Terraform code using the inputs
+* Optionally, run the generated Terraform code:
+  * If Terraform is already installed, the version will be confirmed suitable for use
+	* If Terraform is not installed, or the version installed is not suitable, a new version will be installed into a temporary location
+	* Once Terraform is detected or installed, Terraform plan will be executed
+	* The command will prompt with the outcome of the plan and allow to view more details or continue with Terraform apply
+	* If confirmed, Terraform apply will be run, completing the setup of the cloud account
+
+Usage:
+  lacework cloud-account iac-generate azure [flags]
+
+Aliases:
+  azure, az
+
+Flags:
+      --activity_log                           enable active log integration
+      --activity_log_integration_name string   specify a custom activity log integration name
+      --ad_create                              create new active directory integration (default true)
+      --ad_id string                           existing active directory application id
+      --ad_pass string                         existing active directory application password
+      --ad_pid string                          existing active directory application service principle id
+      --all_subscriptions subscription ids     grant read access to ALL subscriptions within Tenant (overrides subscription ids)
+      --config                                 enable configuration integration
+      --config_name string                     specify a custom configuration integration name
```

Comment:
perhaps the flag should be `configuration` and `configuration_integration_name` to match GCP

Reply:
Yes yes yes 💯

Reply:
Done